plugins/action/common/read_run_map.py

```diff
@@ -39,13 +39,15 @@ def run(self, tmp=None, task_vars=None):
         results = super(ActionModule, self).run(tmp, task_vars)
         results['diff_run'] = True
 
+        fabric_name = task_vars['inventory_hostname']
```

Comment:
I executed a few tests with updating the fabric name under vxlan.global.name, I couldn't see the new run_map file with fabric name prefix. As per this line in the code, I understand we are actually prepending inventory name and not the fabric name with run_map.yml file.

Reply:
@devegupt interesting point you raise; lets discuss in the standup.

Reply:
Right, that's expected with that commit. Before modifying the plugin, the test was to make sure things indeed still worked correctly. I'll modify the code now.